prov/efa/src/efa_base_ep.h

```diff
@@ -33,36 +33,35 @@
 #ifndef EFA_BASE_EP_H
 #define EFA_BASE_EP_H
 
-#include "config.h"
-
-#include <asm/types.h>
-#include <errno.h>
-#include <fcntl.h>
 #include <arpa/inet.h>
-#include <netinet/in.h>
-#include <poll.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <unistd.h>
-#include <assert.h>
-#include <pthread.h>
-#include <sys/epoll.h>
-
-#include <rdma/fabric.h>
-#include <rdma/fi_cm.h>
-#include <rdma/fi_domain.h>
-#include <rdma/fi_endpoint.h>
-#include <rdma/fi_errno.h>
-
 #include <infiniband/verbs.h>
 #include <infiniband/efadv.h>
 
 #include "ofi.h"
-#include "ofi_enosys.h"
-#include "ofi_list.h"
 #include "ofi_util.h"
-#include "ofi_file.h"
+
+#include "efa_av.h"
+
+extern struct fi_provider efa_prov;
+
+#define EFA_WARN(subsys, ...) FI_WARN(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_TRACE(subsys, ...) FI_TRACE(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO(subsys, ...) FI_INFO(&efa_prov, subsys, __VA_ARGS__)
+#define EFA_INFO_ERRNO(subsys, fn, errno) \
+	EFA_INFO(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_WARN_ERRNO(subsys, fn, errno) \
+	EFA_WARN(subsys, fn ": %s(%d)\n", strerror(errno), errno)
+#define EFA_DBG(subsys, ...) FI_DBG(&efa_prov, subsys, __VA_ARGS__)
```

Comment:
Echoing my comment from #8416; move these back to `efa.h`

Reply:
fixed